core/src/test/java/org/frankframework/parameters/DateParameterTest.java

```diff
@@ -490,6 +491,40 @@ public void testParameterFromDateToDate() throws Exception {
 		assertEquals(date, result);
 	}
 
+	@Test
+	public void testTimeParameterWithTimePatternWithoutFormatPattern() throws Exception {
+
+		// Arrange
+		TimeProvider.setTime(LocalDateTime.of(2025, 3, 5, 11, 12, 55));
+		DateParameter parameter = new DateParameter();
+		parameter.setName("time");
+		parameter.setFormatType(DateFormatType.TIME);
+		parameter.setPattern("{now}"); // Now no longer necessary to specify "time" as part of the pattern when type is TIME.
```

Comment:
Als je een DateParam met type "time" wou maken was het nodig om daar een pattern bij te zetten met `{now,time}`. Anders kwam er altijd een parse-exception ergens. Voor een type "date" was het altijd genoeg om te zeggen `{now}`.

Dat vond ik Heel Erg Stom ™️ 

Nu hoeft dat niet meer. Als het een `time` type is dan is het genoeg om `{now}` in het pattern te zetten.